sklearn/tests/test_public_functions.py

```diff
@@ -121,6 +121,7 @@ def _check_function_param_validation(
     "sklearn.metrics.zero_one_loss",
     "sklearn.model_selection.train_test_split",
     "sklearn.svm.l1_min_c",
+    "sklearn.metrics.hamming_loss",
```

Comment:
Please move it to preserve alphabetic order

Reply:
Okay thanks for the comment, I will do it soon !